### PR TITLE
[improvement](statistics)Set min max to NULL when collect stats with sample.

### PR DESCRIPTION
**Pull request by Jibing-Li**
1. To avoid misleading of inaccurate min max stats, set the stats value to NULL while using sample to collect stats.
2. Fix NDV_SAMPLE_TEMPLATE typo, it shouldn't contain row count related contents.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

**Review by Kikyou1997 — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java [hunk 1]
```diff
@@ -239,6 +238,22 @@ protected String getDataSizeFunction(Column column) {
         return "COUNT(1) * " + column.getType().getSlotSize();
     }
 
+    protected String getMinFunction() {
+        if (tableSample == null) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/BaseAnalysisTask.java:242]: Better add comment in code to explain why

**Comment by Jibing-Li** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**